### PR TITLE
Fix #148

### LINKED ISSUE #148: Improve the readme
Currently readme.md only mentions that this bot identifies missing issue number.

It actually does more now:
- identifies missing misc/news.d file
- automatically links bpo
- automatically closes invalid pr
- automatically labels documentation prs with `type-documentation` label
- copies over labels from master PR to backport PRs
- etc

It would be great if the readme can be updated with these information.

### PR DESCRIPTION
**Pull request by ssd71**
Please review the changes

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: README.md [hunk 1]
```diff
@@ -4,13 +4,25 @@
...
 If no b.p.o issue number is found the status fails and the
 "Details" link points to the relevant
 [section of the devguide](https://devguide.python.org/pullrequest/#submitting).
-If an issue number is found then the "Details" link points to the
-issue itself, making it easier to navigate from PR to issue.
+- ### Links to bugs.python.org
+If an issue number is found then the "Details" link points to the relevant issue 
+itself, making it easier to navigate from PR to issue.
+- ### Identifies missing news entry
+If no misc/news.d entry is found or news entry is formatted incorrectly
```
Comment (lysnikolaou) [inline README.md]:
```suggestion
If no `Misc/NEWS.d` entry is found or the news entry is formatted incorrectly 
```

File: README.md [hunk 2]
```diff
@@ -4,13 +4,25 @@
...
 "Details" link points to the relevant
 [section of the devguide](https://devguide.python.org/pullrequest/#submitting).
-If an issue number is found then the "Details" link points to the
-issue itself, making it easier to navigate from PR to issue.
+- ### Links to bugs.python.org
+If an issue number is found then the "Details" link points to the relevant issue 
+itself, making it easier to navigate from PR to issue.
+- ### Identifies missing news entry
+If no misc/news.d entry is found or news entry is formatted incorrectly 
+and issue doesn't have the skip news label, the status fails and relevant
```
Comment (lysnikolaou) [inline README.md]:
```suggestion
and the issue doesn't have the `skip news` label, the status fails and a relevant 
```

File: README.md [hunk 3]
```diff
@@ -4,13 +4,25 @@
...
+- ### Links to bugs.python.org
+If an issue number is found then the "Details" link points to the relevant issue 
+itself, making it easier to navigate from PR to issue.
+- ### Identifies missing news entry
+If no misc/news.d entry is found or news entry is formatted incorrectly 
+and issue doesn't have the skip news label, the status fails and relevant 
+description label is added to it.
+- ### Closes invalid PRs
+Closes PRs that try to merge a maintenance branch into the master branch, adds 
+`invalid` label, and posts relevant message.
```
Comment (lysnikolaou) [inline README.md]:
```suggestion
`invalid` label, and posts a relevant message.
```

**Comment by lysnikolaou** [comment]
Other than that, you also need to sign the CLA, which you can do by following [these](https://devguide.python.org/pullrequest/?highlight=CLA#licensing) instructions.

**Pull request merged**